save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt

```diff
@@ -64,9 +68,19 @@ class SandboxController(
     @GetMapping("/list-file")
     fun listFiles(
         @RequestParam userName: String,
-    ): Flux<String> = blockingToMono { sandboxUserRepository.getIdByName(userName) }
-        .flatMapMany { userId -> storage.list(userId, SandboxStorageKeyType.FILE) }
-        .map { it.fileName }
+    ): Flux<SandboxFileInfo> = blockingToMono { sandboxUserRepository.getIdByName(userName) }
+        .flatMapMany { userId ->
+            storage.list(userId, SandboxStorageKeyType.FILE)
+        }
+        .flatMap {
+            Flux.zip(
```

Comment:
why `Flux.zip`? why not `Mono.zip`

Reply:
Flux of single object -- Mono is more obvious here for me

Reply:
Isn't it a list of objects?
`/list-file` should return a list of files

Reply:
yes, it's a list. but
`Flux.zip(it.toMono(), storage.contentSize(it))` is always a single element

Reply:
I thought it works just like python's zip here...

Reply:
I got it, fixed